src/plugins/intel_npu/src/plugin/npuw/partitioning/partitioning.cpp

```diff
@@ -1327,14 +1354,17 @@ void Partitioner::matchParameters(const std::string& func_name) {
 
     // Now walk other submodels and match parameters with the same key
     // (yes, including the first one)
-    for (auto&& call : model_group) {
+    for (std::size_t call_id{}; call_id < model_group.size(); ++call_id) {
```

Comment:
Please use `= 0` instead of `{}`

Reply:
I don't remember why I used this, I have thought that it can save me from warnings of type-casting errors (like `int64_t i = 0;` but we might need `int64_t i = 0l`), but I might be wrong here

Reply:
Fixed